### PR TITLE
add impact form in EN

### PR DESCRIPTION
**Pull request by kelimuttu**

### CODE REVIEW

**Comment by kelimuttu** [comment]
@MichaelKohler do you have access to the feedback form response to check if the form is working or not?

File: _includes/formcontent.html [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+  </div>
+  <button id="form-submit" type="submit" class="btn btn-default">Submit</button>
+</form>
+
+<div id="input-feedback" class="center"> 
+  <h2>Thank you!</h2>
+  <p>Your answer has been submitted.</p>
+</div>
+
+<iframe src="#" id="no-target" name="no-target" style="visibility:hidden"></iframe>
```
Comment (MichaelKohler) [inline _includes/formcontent.html]: what is this iframe for?

File: _includes/head.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
   <link rel="shortcut icon" href="/assets/favicon.ico">
   <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
   <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/assets/css/datepicker/bootstrap-datepicker3.min.css">
```
Comment (MichaelKohler) [inline _includes/head.html]: do I see it correctly that this is the only css import? Then all other files could be deleted (such as standalone, and the non-minified version)

File: assets/js/bootstrap-datepicker.js [hunk 3]
```diff
@@ -0,0 +1,2096 @@
+/*!
```
Comment (MichaelKohler) [inline assets/js/bootstrap-datepicker.js]: the minified version shows which version it is based on, so I don't think we need the full-version here for reference.

File: assets/js/impactform.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            q2ID + "=" + inputq2 + "&" + 
+            q3IDm + "=" + month + "&" + 
+            q3IDd + "=" + day + "&" + 
+            q3IDy + "=" + year + "&" + 
+            q4ID + "=" + inputq4 + "&" + 
+            q5ID + "=" + inputq5 + "&" + 
+            q6ID + "=" + inputq6 + "&" + 
+            q7ID + "=" + inputq7 + submitRef);
+
+        console.log(submitURL);
```
Comment (MichaelKohler) [inline assets/js/impactform.js]: please remove :)

**Comment by MichaelKohler** [comment]
yes, I have access to the form, will test it out in a few minutes

**Comment by MichaelKohler** [comment]
Does it work for you locally? I get the following error:

screen shot 2017-05-20 at 15 31 38

This also makes the form not be submitted. Otherwise it looks good.

File: _includes/formcontent.html [hunk 5]
```diff
@@ -0,0 +1,476 @@
...
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="Web VR">
+        Web VR
+      </label>
+    </div>
+  </div>
+  <div class="form-group required">
+    <label class="control-label">Who was the main Rep who organized this event?</label>
+    <select class="form-control" id="input-q2" name="q2">
```
Comment (MichaelKohler) [inline _includes/formcontent.html]: at some point we will need to make this dynamic. For now I'm okay with it.

File: _includes/formcontent.html (hunk 1 above)
Comment (kelimuttu) [inline _includes/formcontent.html]: That's from the the sample code in the repo. I will delete it.

File: _includes/head.html (hunk 2 above)
Comment (kelimuttu) [inline _includes/head.html]: you're right. (:

File: assets/js/impactform.js (hunk 4 above)
Comment (kelimuttu) [inline assets/js/impactform.js]: Aaah, right..

File: _includes/formcontent.html (hunk 5 above)
Comment (kelimuttu) [inline _includes/formcontent.html]: Yeah, that's the idea after this first version.

Comment (MichaelKohler) [inline _includes/formcontent.html]: Don't bother with this for now, as I'm working on the backend to actually display some statistics on the website itself, I can just do another endpoint for this, as I already have all that information anyway.

Comment (kelimuttu) [inline _includes/formcontent.html]: Cool. I'll stick with this for now then.

File: assets/js/impactform.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
+$('#input-feedback').hide();
+
+$('[data-toggle="datepicker"]').datepicker({
+    todayHighlight: true,
+    dateFormat: 'dd/mm/yyyy' 
+});
+
+    $('#input-form').one('submit',function(){
```
Comment (MichaelKohler) [inline assets/js/impactform.js]: didn't see that before, sorry.. can you move this to the left so we use the same indentation for everything?

Comment (kelimuttu) [inline assets/js/impactform.js]: Fixed.

**Comment by kelimuttu** [comment]
I put the iframe back as it actually used to throw a page after the form submitted.

**Comment by MichaelKohler** [comment]
Kiki will follow up in a seperate PR. Merging..

**Pull request merged**